docs/modules/ROOT/pages/development.adoc

```diff
@@ -29,9 +29,33 @@ Do 4 steps:
      e.g. "Add new `Department/Name` cop"
   4. Run `bundle exec rake changelog:new` to generate a changelog entry
      for your new cop.
-
 ----
 
+=== Choose a Name
+
+Pick a department. See the xref:cops.adoc[list of existing departments].
+
+Come up with a self-explanatory name.
+
+Name the cop to explain the offense it detects, e.g. `ExtraSpacing`.
+
+Use a leading noun instead of a verb, e.g. use `ArrayAlignment` instead of `AlignArray`.
+
+Make it easy to understand, e.g. use `IndentationStyle` instead of just `Tab`.
+
+Be specific, e.g. use `DuplicateHashKey` instead of just `DuplicateKey`.
+
+Stick to commonly used terms, e.g. use `RedundantPercentI` instead of `RedundantPercentSymbolArray'.
+
+Use correct terms, e.g. arguments in a method call, and parameters in a method signature.
+
+Refer to lines with no symbols as "empty", not "blank", e.g. `LeadingEmptyLines` instead of `LeadingBlankLines`.
+
+Prefer "redundant" to "unneeded", e.g. `RedundantSelf` instead of `UnneededSelf`.
+
+See the https://github.com/rubocop/rubocop/blob/master/config/obsoletion.yml:["renamed" section of `config/obsoletion.yml`]
+for good and bad examples (old name is on the left, new name on the right).
```

Comment:
The link on line 56 seems to be broken because it's including the colon (at least in the github link you added in your comment).

Reply:
Actually looks like the [colon shouldn't be there](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#links)

```suggestion
See the https://github.com/rubocop/rubocop/blob/master/config/obsoletion.yml["renamed" section of `config/obsoletion.yml`]
for good and bad examples (old name is on the left, new name on the right).
```

Reply:
My bad! Thanks for the pointer.